clients/react/src/styles/aim-console.css

```
@@ -417,6 +417,859 @@
+  height: 2px;
+  border-radius: 1px;
+  background: var(--ochre);
+  opacity: 0.55;
+}
+.aim-console .ac-tick.dr:hover,
+.aim-console .ac-tick.cl:hover {
+  left: 1px;
+  right: 1px;
+  height: 5px;
+  opacity: 1;
+  box-shadow: 0 0 6px currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)